test/encoder_test.rb

```diff
@@ -83,20 +83,20 @@ class EncoderTest < Test::Unit::TestCase
 
     should 'handle utf8 strings' do
       bert = [131, 109, 0, 0, 0, 5, 195, 169, 116, 195, 169].pack('C*')
-      assert_equal bert, BERT::Encoder.encode("été")
+      assert_equal bert, BERT::Encoder.encode("été", 0)
```

Comment:
Why do we need this extra `0` param? Does it break how we use the gem in trader?

Reply:
It was introduced to differentiate float encoding some time ago. It defaults to 0 when using `BERT.encode` directly which we do in trader. This argument is actually required when using `Encoder#encode`.

Reply:
thanks!